<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMatch</class>
 <widget class="QDialog" name="CreateMatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Match</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Match Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="matchNameEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Select Map:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="mapBox">
     <item>
      <property name="text">
       <string>Aztec</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dust</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nuke</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Select Team:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="teamBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>Terrorist</string>
     </property>
     <item>
      <property name="text">
       <string>Terrorist</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Counter-Terrorist</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
